.gitignore

```diff
@@ -5,6 +5,7 @@
 
 # Node Modules
 node_modules/
+.npm/
```

Comment:
This prevents a build issue where the github action is formatting the node modules and committing the formatted changes to the repository.